.prettierignore

```diff
@@ -4,5 +4,5 @@ artifacts
 cache
 coverage*
 gasReporterOutput.json
-network.d.ts
+*.d.ts
```

Comment:
prettier was fixing ignored files, this saves the it from doing the unnecessary work.